iroha-lib/model/generators/QueryGenerator.cpp

```diff
@@ -74,73 +72,73 @@ std::shared_ptr<iroha::protocol::Query> QueryGenerator::generateGetAccountTransa
     return query;
 }
 
-std::shared_ptr<iroha::protocol::Query> QueryGenerator::generateGetAccountAssetTransactions(const std::string& accountId,
+std::shared_ptr<iroha::protocol::Query> QueryGenerator::generateGetAccountAssetTransactions(const std::string& account_id,
                                                                                             uint64_t counter,
-                                                                                            const uint64_t createdTime,
+                                                                                            const uint64_t created_time,
                                                                                             const std::string& assetId)
 {
-    auto query = generateQuery(accountId, counter, createdTime);
+    auto query = generateQuery(account_id, counter, created_time);
     auto queryPayload = query->mutable_payload()->mutable_get_account_asset_transactions();
-    queryPayload->set_account_id(accountId);
+    queryPayload->set_account_id(account_id);
     queryPayload->set_asset_id(assetId);
     return query;
 }
 
-std::shared_ptr<iroha::protocol::Query> QueryGenerator::generateGetTransactions(const std::string& accountId,
+std::shared_ptr<iroha::protocol::Query> QueryGenerator::generateGetTransactions(const std::string& account_id,
                                                                                 uint64_t counter,
-                                                                                const uint64_t createdTime,
-                                                                                const std::vector<std::string>& transactionHashes)
+                                                                                const uint64_t created_time,
+                                                                                const std::vector<std::string>& transaction_hashes)
 {
-    auto query = generateQuery(accountId, counter, createdTime);
+    auto query = generateQuery(account_id, counter, created_time);
     auto queryPayload = query->mutable_payload()->mutable_get_transactions();
-    std::for_each(transactionHashes.begin(),
-                  transactionHashes.end(),
+    std::for_each(transaction_hashes.begin(),
+                  transaction_hashes.end(),
                   [&queryPayload](auto tx_hash)
     {
         auto adder = queryPayload->add_tx_hashes();
-//        *adder = tx_hash.to_hexstring();    // TODO
+        //        *adder = tx_hash.to_hexstring();    // TODO
```

Comment:
?

Reply:
Todo resolved. Added new util function. String to hex.